spec/system/admin/tos_banner_spec.rb

```diff
@@ -6,9 +6,14 @@
   include AuthenticationHelper
 
   let(:admin_user) { create(:admin_user, terms_of_service_accepted_at: nil) }
+  let(:test_file) { "Terms-of-service.pdf" }
+  let(:pdf_upload) do
+    Rack::Test::UploadedFile.new(Rails.public_path.join(test_file), "application/pdf")
+  end
 
   before do
     Spree::Config.enterprises_require_tos = true
+    TermsOfServiceFile.create!(attachment: pdf_upload)
```

Comment:
I would have expected that you could pass the local path in directly here, but is that not the case?

Reply:
No, it expects an upload which contains the file type.

Reply:
We could create a factory for this.